owlkettle/gtk.nim

```diff
@@ -365,7 +366,42 @@ proc g_value_unset*(value: ptr GValue)
 proc g_idle_add_full*(priority: cint, fn: GSourceFunc, data: pointer, notify: GDestroyNotify): cuint
 proc g_timeout_add_full*(priority: cint, interval: cuint, fn: GSourceFunc, data: pointer, notify: GDestroyNotify): cuint
 proc g_strdup*(str: cstring): pointer
+proc g_free*(str: pointer)
+{.pop.}
+
+type OwnedGtkString* = distinct cstring
+  # Forward declarations are needed as otherwise the line `cStr.cstring.isNil` 
```

Comment:
Added this comment because imo the reason we do this forward declaration is not straightforward and should be explained.